Manager.php

```diff
@@ -1 +1,41 @@
 <?php
+
+use clean_work_design\ManagerAbstractClass;
+
+class Manager extends ManagerAbstractClass {
+    protected function chooseEmployees(int $empNeeded)
+    {
+        $conn = DB::connect();
+        $query = "SELECT * FROM employees ";
+        $result = mysqli_query($conn,$query);
+        if (!$result) {
+            die("Error :" . mysqli_error());
+        } else {
+            $row = mysqli_fetch_all($result);
+            while ($row == $empNeeded) {
```

Comment:
chỗ này ko cần in ra màn hình nè, ông xóa giúp tui nha

Reply:
ok